build.cake

```diff
@@ -140,12 +140,12 @@ Task("Run-Tests")
     .IsDependentOn("DogfoodBuild")
     .Does(() =>
 {
-     var settings = new DotNetCoreTestSettings
-     {
+    var settings = new DotNetCoreTestSettings
```

Comment:
A lot of whitespace changes here makes this diff pretty noisy!

Reply:
Should I revert the whitespaces I fixed, or keep the changes?